app/assets/stylesheets/application.scss

```diff
@@ -41,6 +42,46 @@
     }
   }
 
+  // We might think about adding this to govuk_frontend_toolkit if other
+  // apps will also be using it.
+  .get-started {
+    .button {
```

Comment:
Is it possible to make clearer what this is doing differently to the included button mixin?